l10n/de/browser/chrome/browser/preferences/privacy.dtd

```diff
@@ -104,7 +104,11 @@
 
 <!ENTITY  clearOnCloseSettings.label     "Einstellungen…">
 <!ENTITY  clearOnCloseSettings.accesskey "E">
-<<<<<<< HEAD
+
+<!ENTITY  browserContainersHeader.label         "Tab-Umgebungen">
+<!ENTITY  browserContainersLearnMore.label      "Weitere Informationen">
+<!ENTITY  browserContainersEnabled.label        "Tab-Umgebungen aktivieren">
+<!ENTITY  browserContainersEnabled.accesskey    "a">
```

Comment:
As we added a lot of string - I prefer to move our block to the end. Will be more easy to merge later.